subpackage/BUILD

```diff
@@ -1,16 +1,23 @@
 load("//:openroad.bzl", "build_openroad")
 
 build_openroad(
-    name = "tag_array_64x184",
-    io_constraints = "//:io-sram",
-    abstract_stage = "floorplan",
-    sdc_constraints = "//:constraints-sram",
+    name = "L1MetadataArray",
+    io_constraints = "//:io",
+    macros = ["//:tag_array_64x184"],
```

Comment:
Introduced an example of a `build_openroad()` call referring to a macro from a different package.